apps/patient/src/components/PharmacyCard.tsx

```diff
@@ -119,13 +126,16 @@ export const PharmacyCard = memo(function PharmacyCard({
     >
       <CardBody p={3}>
         <VStack align="start" w="full" spacing={0}>
-          {pharmacy.info ? (
-            <Tag size="sm" colorScheme={INFO_COLOR_MAP[pharmacy.info]}>
-              <TagLeftIcon
-                boxSize="12px"
-                as={pharmacy.info === 'Previous' ? FiRotateCcw : FiStar}
-              />
-              <TagLabel> {pharmacy.info}</TagLabel>
+          {preferred ? (
+            <Tag size="sm" colorScheme="yellow">
+              <TagLeftIcon boxSize="12px" as={FiStar} />
+              <TagLabel> Preferred</TagLabel>
+            </Tag>
+          ) : null}
+          {previous ? (
```

Comment:
can we make this `previous && !preferred` had a chat with @mrado1 about this for the pharmacy search:
https://github.com/Photon-Health/client/blob/71b86c28b522a428d79df0863f5e2d8953868dff/packages/components/src/systems/PharmacySearch/index.tsx#L237

the gist is it's a little long and confusing when both badges are there

Reply:
updated